frontend_tests/puppeteer_tests/drafts.ts

```diff
@@ -40,13 +40,30 @@ async function create_stream_message_draft(page: Page): Promise<void> {
     await page.keyboard.press("KeyC");
     await page.waitForSelector("#stream-message", {visible: true});
     await common.fill_form(page, "form#send_message_form", {
-        stream_message_recipient_stream: "all",
+        stream_message_recipient_stream: "Verona",
```

Comment:
This needed to be updated to "Verona" to be a proper stream view to support the new tests that were added that involve the stream view and not just the drafts view.